src/plugins/waitlist.js

```diff
@@ -31,14 +31,33 @@ export default function waitlistPlugin () {
       }
     }
 
-    function onModAddDj () {
-      // TODO
+    function onModAddDj (ref) {
+      mp.emit('modAddDj', {
+        mod: mp.user(ref.mi) || mp.wrapUser({ id: ref.mi }),
+        user: mp.wrapUser({ username: ref.t }),
```

Comment:
>I suppose the event from plug.dj doesn't include a user ID?

Nope :/

>a user object with only the username isn't that useful since none of the methods would work anyway…

The reason why I added it, was due to @Burkes saying that this way `user.mention()` will work.